config/routes.rb

```diff
@@ -20,7 +20,12 @@
   resources :conferences
   resources :attendances
   resources :contributors, only: :index
+
   resources :status_updates, only: :show
+  namespace :status_updates do
```

Comment:
I'm not too sure about introducting this namespace. It doesn't really group anything and only holds a single controller.

Of course it not communicated anywhere, not is it set in stone, but I like to think the namespaces serve the purpose of grouping authorization for the different user roles in the app.

What do you think about a top-level controller named `StatusUpdateCommentsController` ?